src/rules/latin-var-name-only.js

```diff
@@ -0,0 +1,21 @@
+const notLatinSymbolRegexp = /[^\w$]/;
+
+module.exports = {
+	meta: {
+		type:"problem"
+	},
+	create (context) {
+		return {
+			Identifier (node) {
+				let active = notLatinSymbolRegexp.test(node.name);
```

Comment:
`active` variable name doesn't reflect the real meaning. Please use the name that is more easy to reason about